events-api/src/modules/events/events.controller.ts

```diff
@@ -21,5 +30,82 @@ EventsController.get(
       data: events,
       searchParams,
     });
-  }
+  },
+);
+
+EventsController.get(
+  "/:eventId/tickets",
+  validateEventId(eventsEventIdDtoSchema),
+  async (req, res) => {
+    const eventId = req.params["eventId"]!;
```

Comment:
Don't use bang! operator

Reply:
> Don't use bang! operator

Why? I saw in the StackOverflow ( https://stackoverflow.com/a/43891263 ) and concluded it's ok because it's clear without eventId, it points to another route...

Reply:
StackOverflow is not always give the correct answers. You can have better ways of doing this type of work by having a generic utility function

Reply:
For examples, prefer refer to another PRs